tests/test_training_validation.py

```
@@ -0,0 +1,684 @@
+    @pytest.mark.xfail(
+        strict=True,
+        reason=(
+            "create_training_args defaults masking to 'none' but "
+            "TrainingArguments defaults to 'global'"
+        ),
+    )
+    def test_create_training_args_masking_matches_training_args_default(self):
+        """The two defaults should agree so create_training_args doesn't
+        silently override the GLiNER default.
+        """
+        sig = inspect.signature(_create_training_args_via_classmethod)
```

Ruff (0.15.1)

[error] Local variable `sig` is assigned to but never used

Remove assignment to unused variable `sig`

(F841)